packages/block-editor/src/components/colors/use-colors.js

```diff
@@ -216,48 +217,27 @@ export default function __experimentalUseColors(
 				break;
 			}
 		}
-		return (
-			( needsBackgroundColor || needsColor ) &&
-			withFallbackStyles(
```

Comment:
I don't think `withFallbackStyles` is need anymore since we have direct access to the element.